.gitignore

```diff
@@ -5,3 +5,4 @@
 .idea
 node_modules
 npm-debug.log
+bower_components
```

Comment:
Hmm, that is a very good point. We would have to include `bower_components` in `gh-pages`; I don't think there's any way around that.

There are [ways](https://gist.github.com/cobyism/4730490) to publish a `dist` folder (or whatever you want to call it) to `gh-pages` while keeping all your source files on `master`. It might be good to combine that effort with introducing a JavaScript bundler.

But, that sounds like too much for this PR. Should I just include `bower_components` for now?